operations/commit_queue.go

```diff
@@ -662,18 +663,14 @@ func (p *moduleParams) addModule(ac *legacyClient, rc *legacyClient) error {
 
 func showCQMessageForProject(ac *legacyClient, projectID string) {
 	projectRef, err := ac.GetProjectRef(projectID)
-	if err != nil {
-		grip.Error(errors.Wrapf(err, "unable to retrieve project '%s'", projectID))
-		return
+	if err == nil && projectRef.CommitQueue.Message != "" {
```

Comment:
Since we don't really care about the error now, I would actually have this check be projectRef != nil. That way we can explicitly discard the error and also nil check